src/main/java/duke/ui/Ui.java

```diff
@@ -0,0 +1,135 @@
+package duke.ui;
+
+import duke.DukeException;
+import duke.parser.Parser;
+import duke.task.Task;
+import duke.task.TaskList;
+
+/**
+ * Represents the client facing UI to handle inputs and displays.
+ */
+public class Ui {
+    /**
+     * Adds task to TaskList and displays added task info to user.
+     * @param tasks TaskList
+     * @param task Task to be added
+     */
+    public String addTask(TaskList tasks, Task task) {
+        String res = "";
+        res += "Got it. I've added this task:\n";
+        res += String.format("  %s\n", task.toString());
+        res += String.format("Now you have %d tasks in the list.", tasks.size());
+        return res;
+    }
+
+    /**
+     * Displays bye message.
+     */
+    public String bye() {
```

Comment:
I think you could benefit from using clearer method names in the `Ui` class
```suggestion
public String getByeMessage() {
```